### DISCUSSION TITLE
Migration of CLI to Bun

### DISCUSSION THREAD

**Discussion by RanitMukherjee** [keyshade-xyz/keyshade#368] [Ideas]
Suggest me some updated blogs, also I am considering using this space for consolidating my experience if it's all right...@rajdip-b

**Comment by rajdip-b** [comment]
I think the issue description would be a better place for this, since discussions are generally focussed at community engagement.

As for the resources, I could find these:
- https://blog.logrocket.com/migrating-typescript-app-node-js-bun/
- https://bun.sh/docs/runtime/typescript

**Comment by RanitMukherjee** [comment]
Cool, I will stick to the Issue description only then & thanks for the sharing the resources, I have already gone through the Blog & will be referring the docs as well.

**Reply by rajdip-b** [reply]
Thank you very much! We are looking forward to a successful bun migration!
